lib/auth/webauthncli/prompt.go

```diff
@@ -109,3 +110,44 @@ func (p *DefaultPrompt) PromptCredential(creds []*Credential) (*Credential, erro
 		fmt.Fprintf(p.out, "Invalid user choice: %q\n", numOrName)
 	}
 }
+
+type credentialPicker interface {
+	PromptCredential([]*CredentialInfo) (*CredentialInfo, error)
+}
+
+// ToTouchIDCredentialPicker adapts a wancli credential picker, such as
+// LoginPrompt or DefaultPrompt, to a touchid.CredentialPicker
```

Comment:
Should this be here, or should this be in `touchid`?

Reply:
We need it here to avoid import cycles.